lib/datadog/tracing/sampling/rule_sampler.rb

```diff
@@ -48,6 +48,33 @@ def initialize(
                              end
         end
 
+        def self.parse(rules, rate_limit, default_sample_rate)
+          rules = JSON.parse(rules).map do |rule|
```

Comment:
```suggestion
          custom_rules = JSON.parse(rules).map do |rule|
```

This is a cosmetic suggestion. That way, it leads to less confusion since the `parse` has the `rules` parameter

Reply:
Fair!